pluginwrapper.py

```diff
@@ -1,42 +1,35 @@
-# plugin wrapper code snippets. handled as macros, to ensure that
-# 1. indentation caused by the "function wrapper()" doesn't apply to the plugin code body
-# 2. the wrapper is formatted correctly for removal by the IITC Mobile android app
-
-# putting everything in a wrapper function that in turn is placed in a
-# script tag on the website allows us to execute in the site's context
-# instead of in the Greasemonkey/Extension/etc. context.
+# plugin wrapper code snippets
 
 # a cut-down version of GM_info is passed as a parameter to the script
 # (not the full GM_info - it contains the ENTIRE script source!)
 
 start = """
-function wrapper(plugin_info) {
-// ensure plugin framework is there, even if iitc is not yet loaded
-if(typeof window.plugin !== 'function') window.plugin = function() {};
+var plugin_info = (typeof GM_info === 'undefined') ? {} : (function (s) {
```

Comment:
Actually we should rely on `GM_info` definition, but iOS is not ready: https://github.com/HubertZhang/IITC-Mobile/issues/1